src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -211,6 +212,9 @@ function ReportActionItemMessageEdit(
         // eslint-disable-next-line react-hooks/exhaustive-deps -- this cleanup needs to be called only on unmount
     }, [action.reportActionID]);
 
+    // show the composer quickly after done editing for devices that hide the composer during editing.
+    useEffect(() => () => ComposerActions.setShouldShowComposeInput(true), []);
+
```

Comment:
Shouldn't this be called for iOS only?

Reply:
In this same file, we call this effect in another flow without having any exclusions for device. I diddn't add any additional logical because of that, and because it's a keyboard aware effect it only impacts IOS and Native experiences inherently. I believe this change improves Android composer show time as well ( I can test this to confirm )